### ISSUE TITLE
Can't use it, always shows error

### ISSUE THREAD

**Issue by ffiona** [avocode/avocode#2191]
Auto update failed

---
* **Version**: 3.5.4
* **Sketch plugin version**: 51.2.0
* **Web/Desktop**: Desktop
* **Platform**: darwin
* **Avocode UID**: 85804
---
```
Error: Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The image argument is a canvas element with a width or height of 0.
    at Object._renderTickCanvas (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:3479114)
    at Object._updateTickCanvas (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:3478770)
    at Object.componentWillUpdate (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:3478522)
    at ReactCompositeComponentWrapper._performComponentUpdate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:714:14)
    at ReactCompositeComponentWrapper.updateComponent (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:645:12)
    at ReactCompositeComponentWrapper.receiveComponent (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react-dom/lib/
```

**Comment by vojtatranta** [comment]
Hi @ffiona ,
we are very sorry to hear this.

This bug is known and we release fix today in 3.5.6.

This issue is duplicate of #2183

**Comment by vojtatranta** [comment]
@ffiona we have just released version `3.5.6` where we fixed this problem.

Sorry again for the troubles, happy avocoding :).
